business/openshift_oauth.go

```diff
@@ -0,0 +1,199 @@
+package business
+
+import (
+	"crypto/tls"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strings"
+
+	"github.com/kiali/kiali/config"
+	"github.com/kiali/kiali/kubernetes"
+
+	"strings"
```

Comment:
Strings is listed twice, along with github.com/kiali/kiali/config and github.com/kiali/kiali/kubernetes

Reply:
Weird, it seems like goimports decided to troll me.